config/application.rb

```diff
@@ -28,5 +28,14 @@ class Application < Rails::Application
 
     # Autoloadable lib directories with classes and modules.
     config.autoload_paths += %W(#{config.root}/lib)
+
+    # Spristupnenie konstant
+    config.before_configuration do
+      env_file = File.join(Rails.root, 'config', 'local_env.yml')
+      YAML.load(File.open(env_file)).each do |key, value|
```

Comment:
Ak by sme sa neskor rozhodli spravit cast z tej konfiguracie nepristupnu pre repozitar, tak by bolo treba prepisovat vela kodu. Konfiguracia tohto charakteru by mala byt v suboroch YAML, aj ked je pravda, ze je to uz na hrane. @mlabaj, co si myslis?

Pripajam este link:
https://practicingruby.com/articles/configurable-applications-1